docs/events.md

```diff
@@ -0,0 +1,220 @@
+# Events specification
+
+This document all the data (and format) which gets send from Atom editor to the GitHub analytics pipeline.
+
+## Type of events
+
+There are 3 different types of events:
+
+* **Counters**: Used to log the number of times a certain event happens. They don't contain any metadata or additional fields.
+* **Timing events**: Used to log duration of certain actions.
+* **Standard events**: Used to log any other action.
+
+## Counters
+
+These events are used to count how many times a certain action happens. They don't hold any metadata and they only log the name of the counter and the number of times it was incremented.
+
+Currently Atom core is not logging any counter event, but the [GitHub package](https://github.com/atom/github) is using counters to log things like the number of created PRs.
```

Comment:
did you want the events logged by `atom/github` to be listed in detail here?

Reply:
Good question 😄 I'm not sure about it...

it would be good to have all the events in a single place but as you mention in the other comment it's going to be hard to keep the docs updated, so if we have also events from other repos it's going to be even harder 😟

What do you think?